python/config_settings/transition.bzl

```
@@ -74,7 +81,7 @@ _COMMON_ATTRS = {
     ),
     "target": attr.label(
         executable = True,
-        cfg = _transition_python_version,
+        cfg = "exec",
```

This needs to be cfg=target, otherwise it could build for a different platform that what it'll run on. e.g., with remote execution, it might build for linux, while locally you build and run a test for mac.

```suggestion
        cfg = "target",
```

---

Of course... addressed in 6896f15.